### ISSUE TITLE
[Meta] Several problems with archived meetings

### ISSUE THREAD

**Issue by MSoeb** [OpenSlides/openslides-client#3136]
**Description:** In an edge case, archived Meetings are not shown in the meeting list. To see the affected archived meetings again, the user needs to open the associated committee of the meeting or the account list. After this, the meeting is visible until the user closes or reloads the website. (Issue 1)

Additionally, the affected archived meetings didn't load directly in the committee detail view. First the list of not affected meetings is loaded and shortly after, the affected meetings are loaded. ( Issue 2)

Furthermore the action Archive triggers a bug, which makes the archived meeting invisible until page reload. (Issue 3)

**Example - Archived meeting C is not visible in meeting list:**
![image](https://github.com/OpenSlides/openslides-client/assets/105491030/8e236eac-342a-49df-8f07-ed5272b27694)

**After opening the committee detail view meeting C is visible:**
![image](https://github.com/OpenSlides/openslides-client/assets/105491030/22cf200a-8379-4366-845c-1f67c049c7c0)

**Reproduction:**
1. open committes > create a committee
2. create two meetings (A and B) in the committee
3. Archive meeting B 
4. Create a new meeting by duplicate archived meeting B. > a new active meeting should now exist > relabel it's name with something like meeting C to identify it easier.
5. archive meeting C > > meeting becomes invisible until page reload (Issue 3)
6. head to Dashboard calender view > reload page with Strg+R
7. open the meeting list and search for meeting C > should not be visible (issue 1)
8. open committee list > open the created committee detail view. the list should now load first the unaffected meetings and after it meeting C. (If you didn't see the first time. Reload the page again.)

What should happen:

- [ ] Issue 1 - Most important: Archived meetings should always be visible even if they are duplicated from an archived meeting.
- [ ] Issue 2: The meeting list in a committee detail view should load all meetings smoothly without a clear delay.
- [ ] Issue 3: Archiving a meeting should not lead to an invisible meeting.

**Comment by luisa-beerboom** [comment]
If one does a thorough datastore check (via `/info` page) after reproducing this sequence, the result will be that the datastore is corrupt and that the newly archived meeting's `is_archived_in_organization_id` field is correctly set to `1`, but the counter-relation is not set.

TLDR: Seems to be primarily a backend problem

**Comment by luisa-beerboom** [comment]
The problem may be that the copy meeting is created with both `is_archived_in_organization_id` and `is_active_in_organization_id` set to `1` at the same time.
